src/assets/courses/rosters.json

```diff
@@ -1 +1 @@
-["WI17","SP17","SU17","FA17","WI18","SP18","SU18","FA18","WI19","SP19","SU19","FA19","WI20","SP20","SU20","FA20","WI21","SP21","SU21","FA21","WI22","SP22","SU22","FA22","WI23","SP23"]
+["WI17","SP17","SU17","FA17","WI18","SP18","SU18","FA18","WI19","SP19","SU19","FA19","WI20","SP20","SU20","FA20","WI21","SP21","SU21","FA21","WI22","SP22","SU22","FA22","WI23","SP23","SU23"]
```

Comment:
I'm not sure how this was added, but the fact that `FA23` isn't in here makes me think that we manually added `SU23` in, either by hard-coding or running some script.

Reply:
Yeah this is added by the `courses-gen` script, which I just ran. However the frontend tests are failing on the branch where I did because the courses json and the course data collection are out of sync. We'll have to merge #820 and populate the course data collection with the FA23 roster to fix this.